src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,105 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] list;
```

Comment:
I think it's better to call `elements `or something like this, maybe list for this task it's not the best name))
```suggestion
    private Object[] elements ;
```